docs/benchmark_configuration/configurations.md

```diff
@@ -113,6 +113,8 @@ Arguments will hame following effect:
 * *args* - VM arguments that will be given to server
   * jboss.node.name - specifies a server node name
   * jboss.socket.binding.port-offset - specifies offset for server port binding, making sure servers do not compete for same ports on localhost
+* *serverng* - true/false. If true server-zip will need to be a serverng zip file   
+* *rest-server* - Infinispan server which will be used to retrieve cluster information via Rest V2 API
```

Comment:
We can assume if rest-server is not empty serverng will be used instead of eap/wildfly server, I just added serverng boolean parameter to make easier to understand which server will be using.
Should I remove serverng parameter?

Reply:
I would be for that, and just improving the documentation message to indicate this. Also is this just a URL to the REST endpoint? Is there any way to determine this programmatically instead of manually entering it?